src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -172,7 +196,41 @@ export class LdPagination {
     this.calculateSliderContent()
   }
 
+  componentDidLoad() {
+    if (this.wrapperRef) {
```

Comment:
Why don't you use an inline event listener? You wouldn't need to take care of removing the listener.

Reply:
Because this does only work for `transitionend` but not `transitionstart`, and also the unit tests seem to not work properly with the event handler props.